docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN CMAKE_VERSION=3.13.4 && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SHA256} && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SHA256}.asc && \
     wget --quiet ${CMAKE_URL}/${CMAKE_SCRIPT} && \
-    gpg --recv-keys ${CMAKE_KEY} && \
+    gpg --keyserver keys.gnupg.net --recv-keys ${CMAKE_KEY} && \
```

Comment:
My experience is that retrieving keys from server will occasionally fail regardless.  On ArborX, at some point we got frustrated we the rate of failure and I [created a keydump](https://github.com/arborx/ArborX/blob/5d53ad63f82acca72279b105c3632569d75c7d78/docker/Dockerfile#L26-L32).  On Kokkos, we specify [yet another key server](https://github.com/kokkos/kokkos/blob/c836655e8236a57cf1522b966e5fa5777899d319/scripts/docker/Dockerfile.clang#L21) and so far we accept that we sometimes will need to relaunch CI builds.